docs/structured-streaming-programming-guide.md

```diff
@@ -910,35 +923,37 @@ Here is the compatibility matrix.
   </tr>
   <tr>
     <td colspan="2" valign="middle"><br/>Queries without aggregation</td>
-    <td>Append</td>
+    <td>Append, Update</td>
```

Comment:
nit: it doesn't support `update`. See https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/UnsupportedOperationChecker.scala#L76